src/components/Navbar/Navbar.css

```diff
@@ -18,6 +18,18 @@
 		rgb(72 97 118) 74%,
 		rgb(74 101 122) 86%,
 		rgb(79 106 127) 100%
+	); */
+	background: linear-gradient(
+		205deg,
+		rgb(67 92 112) 7%,
+		rgb(67 92 112) 20%,
+		rgb(67 92 112) 30%,
+		rgb(68 91 111) 46%,
+		rgb(67 92 112) 53%,
+		rgb(68 91 112) 65%,
+		rgb(67 92 112) 74%,
+		rgb(67 92 112) 86%,
+		rgb(67 92 112) 100%
```

Comment:
Suggestion : color variables where ever possible

Reply:
these color range is not fixed yet